test/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd ./test
```

Comment:
is it started from the folder above? Maybe this:

```suggestion
cd "$(dirname "$0")"
```

Reply:
I removed the shell script and moved the cd command back in the yml